apps/gallery/index.html

```diff
@@ -59,26 +59,32 @@
   </head>
 
   <body role="application" class="theme-media">
-    <section role="region" id="thumbnail-list-view" class="skin-dark">
-      <!-- Header only shown on tablet view  -->
-      <gaia-header id="thumbnails-list-top">
+    <section role="region" id="thumbnail-views" class="skin-dark">
```

Comment:
I'd appreciate a comment above this line saying something like:

  This section holds the thumbnails list and the headers and footers for all views that display the thumbnails